docs/Installation.rst

```diff
@@ -0,0 +1,114 @@
+.. role:: underline
+    :class: underline
+.. role:: bolditalic
+    :class: bolditalic
+
+********************
+Building Stand-Alone
+********************
+
+=====================
+Software Requirements
+=====================
+
+Before installing the UPP code, it is necessary to ensure that you have the required libraries
+available on your system. These libraries include:
+
+  - The external NCEP libraries
+    https://github.com/NOAA-EMC/NCEPLIBS-external
+
+  - The NCEP libraries
+    https://github.com/NOAA-EMC/NCEPLIBS
+
+An introduction of each can be found in their respective top level :bolditalic:`README.md` files.
+Detailed instructions for building the libraries on various platforms can be found in the **NCEPLIBS-external/doc** directory.
+
+Certain machines do have the NCEP libraries in a pre-installed location for use to build UPP. Paths to
+these pre-installed libraries are available on the
+`UFS-SRW wiki <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_
+and include platform name and compiler version.
+
+============================
+Obtaining and Installing UPP
+============================
+
+Building and running UPP V9.0.0 has been tested on the following platforms using pre-configured libraries.
+
++---------------+----------------------+
+| System        | Compiler and Version |
++===============+======================+
+| NCAR Cheyenne | Intel 19.1.1         |
+|               +----------------------+
```

Comment:
UPP is also pre-built for GNU 10.1.0: /glade/p/ral/jntp/UPP/UPP_v9.0.0_gnu-10.1.0

Reply:
Thanks @mkavulich. Added.